phpunit/class-wp-rest-url-details-controller-test.php

```diff
@@ -124,11 +81,13 @@ public function test_get_items() {
 		$response = rest_get_server()->dispatch( $request );
 		$data     = $response->get_data();
 
-		// Note the data in the subset comes from the fixture HTML returned by
-		// the filter `pre_http_request` (see this class's `setUp` method).
-		$this->assertArraySubset(
+		/*
+		 * Note the data in the subset comes from the fixture HTML returned by
+		 * the filter `pre_http_request` (see this class's `setUp` method).
+		 */
+		$this->assertSame(
 			array(
-				'title'       => 'Example Website &mdash; - with encoded content.',
+				'title'       => 'Example Website — - with encoded content.',
```

Comment:
Is it intentional to have both types of "dash" here?

Reply:
Left the original fixture as may used elsewhere. Though it looks weird, it does show the entity being converted back into its character while the other character is not touched.